### PR TITLE
kernelci.org: update admin documentation

### PR DESCRIPTION
**Pull request by JenySadadia**
Update documentation for getting admin API token and creating user accounts as per the latest changes in the API and `kci` command syntax.

### CODE REVIEW

**Comment by gctucker** [comment]
LGTM, I'll check it looks OK on the staging website later today.

File: kernelci.org/content/en/docs/admin/api.md [hunk 1]
```diff
@@ -7,39 +7,36 @@ description: "KernelCI API"
...
+  "groups": [
+    {
+      "id": "6499aa9da02fef8143c1feb0",
+      "name": "admin"
     }
+  ]
 }
-$ kci user get_token --username=<your-username> --scopes=admin
+```
+
```
Comment (gctucker) [inline kernelci.org/content/en/docs/admin/api.md:30]: It's missing some \`\`\` here or the line above shouldn't have one.  See the glitch on the rendered page:
https://static.staging.kernelci.org/docs/admin/api/

**Review by gctucker — CHANGES_REQUESTED** [review]
A small issue with code blocks.

File: kernelci.org/content/en/docs/admin/api.md (hunk 1 above)
Comment (JenySadadia) [reply kernelci.org/content/en/docs/admin/api.md]: Thanks for pointing it out. Fixed it.

**Comment by JenySadadia** [comment]
Looks OK on staging: https://static.staging.kernelci.org/docs/admin/api/

**Pull request merged**